README.md

```diff
@@ -1 +1,56 @@
-NL-Design-System
+# NL-Design-System
+
+The NL Design Skeleton application is designed for rapid application testing and prototype development on the NL Design System. It provides a basic skeleton application with full NL Design system functionality that any developer can easily extend, view locally and deploy to an online environment for demonstration purposes. The main benefits are:
+
+- Development and (online) demonstration of prototypes without the need of a server.
+
+- An out-of-the-box basic application that doesn't require configuration or setup and can be extended immediately.
+
+This NL-Design-system repository is a customizable and straightforward component library to help make Progressive Web Applications faster, attractive, and more accessible with React.
+
+## Installation
+
+NL-Design-System is available as an NPM package.
+
+``// with npm
+npm install @conductionnl/nl-design-system
+
+// with yarn
+yarn @conductionnl/nl-design-system
+``
+
+# Usage
+
+Here's a quick example to get you started, _it's all you need_
+
+```JS
+import  *  as  React  from  "react";
+import Alert from "@conductionnl/nl-design-system";
+
+export  function  Alert(props) {
+
+return (
+
+	<>
+
+		<div style={{ position:  "fixed", left:  "50%", top:  "90px", transform:  "translateX(-50%)" }} id={props.id ? props.id + "Alert" : "Alert"} className={`alert alert-${props.alertClass} utrecht-alert utrecht-alert-${props.alertClass}`}
+role="alert">
+
+			{props.body()}
+
+			</div>
+
+		</>
+
+	);
+
+}
+```
```

Comment:
Dit klopt gedeeltelijk hoe je het nu hebt opgeschreven. Conceptueel werkt een component package (en dus ook deze) als volgt:

**Package (provider) side:**
(1) In de package worden componenten gebouwd, waar variabelen (of in React terms `props`) aan meegegeven kunnen worden;
(2) Deze componenten worden exported;

**Client (consumer) side:**
(3) De package kan geïnstalleerd worden in een code base (m.b.v. e.g. `npm i ...` zoals eerder omschreven);
(4) De individuele componenten kunnen ingeladen worden in bestanden, layouts, componenten, whatever;
(5) De ingeladen componenten kunnen nu d.m.v. `props` customised worden.

Bovenstaand betekent dat de consumer side nooit de code van de componenten hoeft te zien (dat is juist heel het punt, daar hoeven ze niet meer over na te denken), maar deze code toon je nu wel in de _Usage_.

Dus de usage komt eigenlijk neer op het volgende (ik heb even het component `InfoTooltip` genomen, omdat we deze recentelijk netjes in Typescript hebben geschreven):

```TS
// All components come with prop autocompletion shown directly in your IDE

Import { InfoTooltip } from "@conductionnl/nl-design-system"

<InfoTooltip  content={<>Pass any JSX.Element here</>} placement={"bottom"} /> 
```